templates/rhpam75-prod-immutable-kieserver-amq.yaml

```diff
@@ -48,14 +48,14 @@ parameters:
   value: executionUser
   required: false
 - displayName: KIE Server Password
-  description: KIE server password, used to connect to KIE servers. Generated value can be a suggestion to use for thew s2i various (Sets the org.kie.server.pwd system property)
+  description: KIE server password. If this parameter is not set, the password is generated automatically. (Sets the org.kie.server.pwd system property)
```

Comment:
I believe that "the password is automatically generated" sounds better.

Reply:
While it does not seem so to me, I can make this change.